backend/src/main/java/server/SecretHitlerServer.java

```diff
@@ -28,10 +31,10 @@ public class SecretHitlerServer {
     public static final int DEFAULT_PORT_NUMBER = 4040;
 
     // Environmental Variable Names
-    private static final String ENV_DEBUG = "DEBUG_MODE";
+    public static final String ENV_DEBUG = "DEBUG_MODE";
     private static final String ENV_DATABASE_URL = "DATABASE_URL";
 
-    private static boolean DEBUG = System.getenv(ENV_DEBUG) != null;
+    private static boolean DEBUG = System.getProperty(ENV_DEBUG) != null;
```

Comment:
Correct, i can add that do the change aswell.

Reply:
Changed how this was implemented. I added a class `ApplicationConfig` that can hold information that read from environment-variables. And when running the local-variant I just overwrite the value on the static value on the class. This way it will behave as it used to when you run the normal main method, and will use the value from the environment-variable instead of using a java system property.